stdlib/universe/state_tracking_test.go

```diff
@@ -356,5 +370,32 @@ func TestStateTracking_Process(t *testing.T) {
 				},
 			)
 		})
+		t.Run(
+			fmt.Sprintf("%s narrow", tc.name),
```

Comment:
I'd just update the old test to call into the new code. We're not changing the old code so I don't think we need that test anymore.